### DISCUSSION TITLE
Train and Deploy Your Own AI Model with Roboflow, YOLOv5, TensorFlow Lite | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1197] [Q&A]
# Train and Deploy Your Own AI Model with Roboflow, YOLOv5, TensorFlow Lite | Seeed Studio Wiki

Train and Deploy Your Own AI Model with Roboflow, YOLOv5, TensorFlow Lite

https://wiki.seeedstudio.com/Train-Deploy-AI-Model-Grove-Vision-AI/

**Comment by feiticeir0** [comment]
If you're going to try this using your machine (not a google colab)  and using Yolov5-swift, you must create the virtual environment using Python3.10 . 
After the step installing requirements.txt, you must replace numpy-1.24 with:
pip install "numpy<1.24" 

After that, you can train it without any errors.
